controllers/windowsmachine_controller.go

```diff
@@ -279,7 +282,7 @@ func (r *WindowsMachineReconciler) Reconcile(ctx context.Context, request ctrl.R
 			return ctrl.Result{}, nil
 		}
 	} else if *machine.Status.Phase != provisionedPhase {
-		log.V(1).Info("machine not provisioned", "phase", *machine.Status.Phase)
+		log.V(1).Info("machine not provisioned, updating metrics endpoints", "phase", *machine.Status.Phase)
 		// configure Prometheus when a machine is not in `Running` or `Provisioned` phase. This configuration is
 		// required to update Endpoints object when Windows machines are being deleted.
 		if err := r.prometheusNodeConfig.Configure(); err != nil {
```

Comment:
Do we need to do this here now? Given we have moved updation of metrics endpoint for machines being deleted if machine not found ?